add support for .p7m signed files (#39)
demisto/parse-emails: CHANGELOG.md

# Changelog
v0.0.13
* Fixed an issue where signed .p7m files were not recognized.

> * Added the option the pass the file-name.

Reviewer: ```suggestion
* Added the option to pass in the filename.
```
